lib/delocalize.rb

```diff
@@ -1,19 +1,5 @@
-if defined?(Rails::Railtie)
-  require 'delocalize/railtie'
-elsif defined?(Rails::Initializer)
-  raise "This version of delocalize is only compatible with Rails 3.0 or newer"
-end
-
 module Delocalize
   class ParserNotFound < ArgumentError; end
 
   autoload :Parsers, 'delocalize/parsers'
-
-  # this ensures that a subsequent call to `column_for_attribute` will return a column and not nil
-  def self.model_has_column?(model, column_name)
```

Comment:
This is the only method I'm not sure about deleting. See internals for more detail.